micronaut-validation - java 9 modules - java.lang.module.FindException: Unable to derive module descriptor for

### Steps to Reproduce

1. Define Java 9 module in the application
2. Include dependency to `micronaut-validation`
3. Run the application

### Expected Behaviour

Application executes without any problem

### Actual Behaviour

```
Error occurred during initialization of boot layer
java.lang.module.FindException: Unable to derive module descriptor for C:\Users\XXX\.gradle\caches\modules-2\files-2.1\io.micronaut\micronaut-validation\3.2.3\8ecda68304d991450306ab217f62e2fc6316fd64\micronaut-validation-3.2.3.jar
Caused by: java.lang.module.InvalidModuleDescriptorException: Provider class io.micronaut.validation.websocket.WebSocketVisitor not in module
```

This seems to be caused by changes introduced in PR #5666 - the class `io.micronaut.validation.websocket.WebSocketVisitor` was moved to a separate module `http-validation` but it is (and perhaps not only this class) still mentioned in [validation/src/main/resources/META-INF/services/io.micronaut.inject.visitor.TypeElementVisitor](https://github.com/micronaut-projects/micronaut-core/blob/3.2.x/validation/src/main/resources/META-INF/services/io.micronaut.inject.visitor.TypeElementVisitor) file

I'm wondering wheter this can be somehow checked somewhere during a CI/CD pipeline?

### Environment Information

- **Operating System**: Windows 10
- **Micronaut Version:** 3.2.3
- **JDK Version:** 17.0.1

### Example Application

- https://github.com/oujesky/micronaut-validation-module-bug
- just run the app with `./gradlew run` 
- it will fail with the described exception

--- a/validation/src/main/resources/META-INF/services/io.micronaut.inject.visitor.TypeElementVisitor
+++ b/validation/src/main/resources/META-INF/services/io.micronaut.inject.visitor.TypeElementVisitor
@@ -1,5 +1,3 @@
-io.micronaut.validation.websocket.WebSocketVisitor
 io.micronaut.validation.async.AsyncTypeElementVisitor
-io.micronaut.validation.routes.RouteValidationVisitor
 io.micronaut.validation.internal.InternalApiTypeElementVisitor
 io.micronaut.validation.executable.ExecutableVisitor

